examples/ads.amp.html

```diff
@@ -338,42 +414,41 @@ <h2 id="dotandads">DotAndAds masthead</h2>
       data-cid="11"
       data-mpo="ampTest"
       data-mpt="amp-amp-all-all">
+    <div placeholder></div>
+    <div fallback></div>
   </amp-ad>
 
   <h2 id="doubleclick">Doubleclick</h2>
   <amp-ad width=320 height=50
       type="doubleclick"
       data-slot="/4119129/mobile_ad_banner">
+    <div placeholder></div>
+    <div fallback></div>
   </amp-ad>
 
   <h2>Doubleclick with JSON based parameters</h2>
   <amp-ad width=320 height=50
       type="doubleclick"
       data-slot="/4119129/mobile_ad_banner"
       json='{"targeting":{"sport":["rugby","cricket"]},"categoryExclusions":["health"],"tagForChildDirectedTreatment":0}'>
+    <div placeholder></div>
+    <div fallback></div>
   </amp-ad>
 
   <h2>Doubleclick no ad with fallback</h2>
   <amp-ad width=300 height=200
       type="doubleclick"
       data-slot="/4119129/doesnt-exist">
-      <div fallback>
-        <b>Thank you for trying AMP!</b>
-        <p>You got lucky! We have no ad to show to you!</p>
-      </div>
+    <div placeholder></div>
+    <div fallback></div>
```

Comment:
We should probably leave this and the the one below unchanged since they are here to test fallback and placeholder specifically. Or remove these two examples.